redaxo/src/core/lib/util/editor.php

```diff
@@ -42,7 +42,9 @@ public function getUrl($filePath, $line)
 
         $editorUrl = null;
 
-        if (isset($this->editors[$editor])) {
+        // don't provide editor urls for paths containing "://", like "rex://..."
+        // but they can be converted into an url by the extension point below
+        if (isset($this->editors[$editor]) && false !== strpos($filePath, '://')) {
```

Comment:
Hmm d.h. Eine url würde dann platzhalter nicht ersetzt bekommen und der EP müsste %line und %file ersetzen?

Reply:
Die Extensions liefern doch entweder eine ganz andere URL (Backend-URLs), oder im Fall von Developer eine URL zu einer anderen Datei, generiert über einen rekursiven Aufruf von `rex_editor::factory()->getUrl(...)`.

Durch diesen PR ändert sich nichts diesbezüglich. 

Die Platzhalter sind doch in den Editor-URLs, nicht in $filePath (was eben auch eine "URL" sein kann).